src/components/Chat/ChatItem.tsx

```diff
@@ -17,11 +17,14 @@ import Box from '../Box/Box'
 import { ComponentSlotStylesPrepared } from '../../themes/types'
 
 export interface ChatItemProps extends UIComponentProps, ChildrenComponentProps {
+  /** Attach ChatItem to other content. */
```

Comment:
would suggest to omit component class names in the description - as it should just convey general purpose of the prop. Something like: `Controls item's relation to other chat items`